GitHubRepositories.ps1

```diff
@@ -86,6 +86,9 @@ filter New-GitHubRepository
         By default, rebase-merge pull requests will be allowed.
         Specify this to disallow.
 
+    .PARAMETER AllowAutoMerge
+        Specifies whether to allow auto-merge on pull requests.
```

Comment:
Worthwhile clarifying that this is only valid on public repo's.  Something like this:

```suggestion
        Specifies whether to allow auto-merge on pull requests.  Currently only supported for public repositories.
```

Reply:
The condition is not that simple though, see [Automatically merging a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request). GitHub may also change what products they support this option on, so I would leave it out of the documentation, the same as it is not specified for other options that have differing availability across the GitHub products.

Reply:
That's totally reasonable.  Thanks for pointing out the docs on that.  Agreed that we shouldn't detail the complexities for what types of repos it will/will not work on in these docs.